app/Models/Colleagues.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Backpack\CRUD\CrudTrait;
+
+class Colleagues extends Model
+{
+    use CrudTrait;
+    protected $table = 'colleagues_data';
+    public $timestamps = false;
+    protected $fillable = ['name','jobs','subjects','roles','awards','image','category'];
+    protected $hidden = [];
+    //Follows the format of the old website. For example jobs=igazgató|subjects=matematika,technika
+    //|roles:"A Verseghy Diákokért" Alapítvány kuratóriumának elnöke,|awards=Graphisoft-díj (2002.)
```

Comment:
You should add a mutator to upload the image, see https://github.com/Laravel-Backpack/CRUD/issues/1219#issue-293761603 for an example

Reply:
Added mutator.